pkg/controllers/common/constants.go

```diff
@@ -116,6 +116,10 @@ const (
 	TemplateGeneratorMergePatchAnnotation = FederateControllerPrefix + "template-generator-merge-patch"
 
 	LatestReplicasetDigestsAnnotation = DefaultPrefix + "latest-replicaset-digests"
+
+	// NoSyncAnnotation indicates skip syncing. This annotation should only be added manually and temporarily in particular cases.
+	// It works only when workloads have not been propagated to member clusters.
```

Comment:
This line is inaccurate; the annotation also stops any updates from being propagated to member clusters. Consider changing it to `"It will prevent new resources from being dispatched and existing resources from being updated by the sync controller"`.

Reply:
Pls see the comment for the handling of `no-sync` annotation first.